### PR TITLE
Request for adding the PISCES paper

### PR DESCRIPTION
**Pull request by mshahbaz**

### CODE REVIEW

**Comment by mshahbaz** [comment]
Hi Nate, I just added the PISCES paper to the publications.yml file. Please let me know if this looks good. Thanks.

File: _data/publications.yml [hunk 1]
```diff
@@ -33,6 +33,13 @@
     date: '2016-04-09'
     url: 'https://www.sigcomm.org/sites/default/files/ccr/papers/2016/April/0000000-0000002.pdf'
     type: 'journal'
+-
+    title: "PISCES: A Programmable, Protocol-Independent Software Switch"
+    authors: "Muhammad Shahbaz, Sean Choi, Ben Pfaff, Changhoon Kim, Nick Feamster, Nick McKeown, and Jennifer Rexford"
+    venue: "ACM SIGCOMM"
```
Comment (jnfoster) [inline _data/publications.yml]: Please edit this line so it has the full name of the conference with the acronym in parenthesis, like the other entries.

Comment (mshahbaz) [reply _data/publications.yml]: I believe the SIGCOMM conferences are typically written as "ACM SIGCOMM." Please confirm if you want me to expand the SIGCOMM acronym, i.e., ACM Special Interest Group on Data Communication (SIGCOMM)?

Comment (jnfoster) [reply _data/publications.yml]: Let's go with "ACM Conference of the Special Interest Group on Data Communication (SIGCOMM)"

**Comment by mshahbaz** [comment]
Fixed the venue name.

**Comment by jnfoster** [comment]
Thanks!

**Pull request merged**